model/task/task.go

```diff
@@ -233,6 +233,8 @@ type Task struct {
 	// DisplayStatus is not persisted to the db. It is the status to display in the UI.
 	// It may be added via aggregation
 	DisplayStatus string `bson:"display_status,omitempty" json:"display_status,omitempty"`
+	// DisplayStatusCache is persisted to the db, unlike DisplayStatus.
```

Comment:
nit: can we also mention "it's semantically the same as DisplayStatus, but is persisted to the DB.

Reply:
Created DEVPROD-13241 w.r.t. the comment above.